pkg/pillar/cmd/zedagent/handlecertconfig.go

```diff
@@ -127,70 +142,181 @@ func unpublishControllerCert(ctx *getconfigContext, key string) {
 func handleEveNodeCertModify(ctxArg interface{}, key string,
 	configArg interface{}) {
 
-	// XXX TBD
+	ctx := ctxArg.(*zedagentContext)
 	status := configArg.(types.EveNodeCert)
 	log.Infof("handleEveNodeCertModify for %s", status.Key())
+	triggerEveNodeCertEvent(ctx)
 	return
 }
 
 func handleEveNodeCertDelete(ctxArg interface{}, key string,
 	configArg interface{}) {
 
-	// XXX TBD
+	ctx := ctxArg.(*zedagentContext)
 	status := configArg.(types.EveNodeCert)
 	log.Infof("handleEveNodeCertDelete for %s", status.Key())
+	triggerEveNodeCertEvent(ctx)
 	return
 }
 
 // Run a task certificate post task, on change trigger
-func certsTask(ctx *zedagentContext, triggerCerts <-chan struct{}) {
-	iteration := 0
+func controllerCertsTask(ctx *zedagentContext, triggerCerts <-chan struct{}) {
+
 	log.Infoln("starting certs publish task")
-	publishCerts(ctx, iteration)
+	getCertsFromController(ctx)
```

Comment:
I guess we need this now when we do not have a hash sent from the controller. But once the controller sends the hash we could use the trigger (plus saving the controller certs in a file) to avoid any fetches until there is a change. But no need to change it now.

Reply:
ok